kong/cluster_events/strategies/cassandra.lua

```diff
@@ -26,7 +27,7 @@ local mt = { __index = _M }
 function _M.new(db, page_size, event_ttl)
   local self  = {
     cluster   = db.connector.cluster,
-    page_size = page_size or 100,
+    page_size = page_size or constants.DEFAULT_ITERATION_SIZE,
```

Comment:
The `kong.cluster_events` module should be authoritative with regards to this value. We should probably guarantee that the cluster_events strategies always receive a `page_size` argument.

Reply:
Was there by the way a reason for this being `100` instead of `1000`. Should this PR still use `100` instead?